curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/6633b4d3271d5b3aea1c1195.md

```diff
@@ -0,0 +1,172 @@
+---
+id: 6633b4d3271d5b3aea1c1195
+title: Step 13
+challengeType: 20
+dashedName: step-13
+---
+
+# --description--
+
+Now it's time for the graph.
+Create a function `__create_trajectory`.
+
+The function should return a list of strings, where each string is a row of the final graph, you should add a `∙` for each point rapresenting the projectile coordinates.
+
+Important notes: round each coordinate to the nearest integer to have the points to write in the graph, and for the symbol you can use the `PROJECTILE` variable.
+
+For example, for the coordinates from `Projectile(12, 3, 45)`, the output of the method should be as follow:
+
+```py
+[
+    "                  ",
+    "      ∙∙∙         ",
+    "    ∙∙   ∙∙∙      ",
+    "  ∙∙        ∙     ",
+    " ∙           ∙∙   ",
+    "∙              ∙  ",
+    "                ∙ ",
+    "                  ",
+    "                 ∙",
+]
+```
+
+# --hints--
+
+You should have the method.
+
+```js
+({
+    test: () => assert(runPython(
+        `_Node(_code).find_class('Graph').has_function('__create_trajectory')`
+    ))
+})
+```
+
+The method should output the correct output.
+
+```js
+({
+    test: () => assert(runPython(
```

Comment:
I would add another test, otherwise this can happen:
![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/105294544/080525ea-7c46-43da-9fb4-750b0d406df9)